### ISSUE TITLE
Upgraded to unlshd-072e Cross Remote updated to 2.4 - Crash Reboot

### ISSUE THREAD

**Issue by gilgrissom** [leedave/flipper-zero-cross-remote#15]
Upgraded to unlshd-072e
Cross Remote updated to 2.4
The application runs fine but when you try to run a saved command (multiple impulses to 2 different led-light ir-enabled devices) Flipper reboots then...
"Flipper crashed and was rebooted"
"NULL pointer dereference"

unlshd-071e version worked fine

**Comment by leedave** [comment]
Will need some time to setup a device and look into this. Seems like a Unleashed specific error, as so far I have no problems with tests on Official Firmware and Roguemaster. 

It would be good if you could provide your saved *.xr file to help me replicate the error.

**Comment by gilgrissom** [comment]
Thanks for the feedback.
For your convenience, [here the unlshd-072 Release Notes](https://github.com/DarkFlippers/unleashed-firmware/releases/tag/unlshd-072)
Attached the *.xr + *.conf
[xr+conf.zip](https://github.com/leedave/flipper-zero-cross-remote/files/14782951/xr%2Bconf.zip)

**Comment by leedave** [comment]
Ok, doesn't seem to be related to unleashed. I can replicate the error with OFW. 

The IR Format has changed in the last releases, as there is now an option to choose how long each IR signal is sent. I did add code to compensate old files if this data is missing, but somehow on your file it didn't work correctly. 
The easiest workaround is create new command chains or update the files manually. 

I've added to timings to your file and it works now. Any newly created remotes should not have this problem. 

Download the updated files here 
[xr_new.zip](https://github.com/leedave/flipper-zero-cross-remote/files/14785533/xr_new.zip)
